src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,16 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory INSTANCE =
+            new Configuration().configure().buildSessionFactory();
+
+    private HibernateUtil() {
+    }
+
     public static SessionFactory getSessionFactory() {
-        return null;
+        return INSTANCE;
     }
```

Comment:
let's add a method - initSessionFactory where you'll return the 
`new Configuration().configure().buildSessionFactory();`
so, you'll have a method for creating the configuration, therefore you can use it in 7 line and return the instance in the getSessionFactory as it's now.
Also, we can write an INSTANCE variable not in upperCase in this situation, but it's up to you